web/src/pages/apd/activities/overview/HITECHOverview.js

```diff
@@ -0,0 +1,258 @@
+import { FormLabel } from '@cmsgov/design-system';
+import PropTypes from 'prop-types';
+import React, { Fragment, useEffect, useMemo } from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { joiResolver } from '@hookform/resolvers/joi';
+import { connect } from 'react-redux';
+
+import { t } from '../../../../i18n';
+import {
+  setActivityAlternatives,
+  setActivityDescription,
+  setActivityOverview
+} from '../../../../redux/actions/editActivity';
+import FundingSourceForm from './FundingSourceForm';
+import Instruction from '../../../../components/Instruction';
+import NameForm from './NameForm';
+import RichText from '../../../../components/RichText';
+import { selectActivityByIndex } from '../../../../redux/selectors/activities.selectors';
+import { Subsection } from '../../../../components/Section';
+
+import { hitechActivityOverviewNoSCSchema } from '@cms-eapd/common/schemas/activityOverview';
+
+const HITECHOverview = ({
+  activity,
+  activityIndex,
+  setAlternatives,
+  setDescription,
+  setOverview,
+  adminCheck
+}) => {
+  const { activityOverview: { alternatives, description, summary } = {} } =
+    activity;
+
+  const {
+    control,
+    trigger,
+    clearErrors,
+    formState: { errors }
+  } = useForm({
+    defaultValues: {
+      alternatives: alternatives,
+      description: description,
+      summary: summary
+    },
+    resolver: joiResolver(hitechActivityOverviewNoSCSchema)
```

Comment:
Might be a good idea to have a comment here or at the import spelling out SC = Standards and Conditions. What do you think?

Reply:
I am all for airing on the side of being verbose for better code readability- I'll add in a comment to clarify that!

Reply:
I ended up adding a comment right at the source- the schema